src/typescript/frontend/src/components/selects/trade-options/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from "react";
+
+import { DropdownMenuWrapper } from "../dropdown-menu/styled";
+
+import { type TradeOptionsProps } from "../types";
+import {
+  DropdownMenuInner,
+  StyledDropdownMenuClose,
+  StyledDropdownMenuItem,
+} from "../dropdown-menu//components/dropdown-menu-item/styled";
+import { InputNumeric } from "components/inputs";
+import { DEFAULT_MAX_SLIPPAGE } from "const";
+
+import * as SlippageSettings from "../../../utils/slippage";
+
+export const TradeOptions: React.FC<TradeOptionsProps> = ({ onClose, onMaxSlippageUpdate }) => {
+  const [maxSlippage, setMaxSlippage] = useState<bigint>(DEFAULT_MAX_SLIPPAGE);
+  const [maxSlippageMode, setMaxSlippageMode] = useState<SlippageSettings.MaxSlippageMode>("auto");
+  useEffect(() => {
+    const { mode, maxSlippage } = SlippageSettings.getMaxSlippageSettings();
+    setMaxSlippage(maxSlippage);
+    setMaxSlippageMode(mode);
+  }, []);
+  return (
+    <DropdownMenuWrapper>
+      <StyledDropdownMenuClose
+        key="close"
+        disabled={false}
+        onClick={onClose}
+        className="cursor-pointer"
+      >
+        <DropdownMenuInner
+          className="flex"
+          style={{ justifyContent: "end", margin: "0", padding: "0" }}
+        >
+          <div className="med-pixel-text lowercase">x</div>
+        </DropdownMenuInner>
+      </StyledDropdownMenuClose>
```

Comment:
```suggestion
```

Much cleaner without the close here imo, plus it already disappears if you click off of it, like the wallet adapter dropdown. Looks incredible this way too :)